tests/bud.bats

```diff
@@ -3590,15 +3625,17 @@ _EOF
   d1="$output"
   run jq -r '.manifests[1].digest' <<< "$list"
   d2="$output"
-  test -n "$d1"
-  test -n "$d2"
-  test "$d1" != "$d2"
+  assert "$d1" =~ "^sha256:[0-9a-f]{64}\$"
+  assert "$d2" =~ "^sha256:[0-9a-f]{64}\$"
+  assert "$d1" != "$d2" "digest(arm) should != digest(386)"
 }
 
 @test "bud-multiple-platform-no-partial-manifest-list" {
   outputlist=localhost/testlist
   run_buildah 1 bud --signature-policy ${TESTSDIR}/policy.json --platform=linux/arm,linux/amd64 --manifest $outputlist -f ${TESTSDIR}/bud/multiarch/Dockerfile.fail ${TESTSDIR}/bud/multiarch
+  expect_output --substring "error building at STEP \"RUN test .arch. = x86_64"
   run_buildah 125 manifest inspect $outputlist
+  expect_output --substring "reading image .* pinging container registry"
```

Comment:
This is another error message I'm not 100% sure about. `pinging blah blah` is the error we get, and is easily reproduced on my laptop, I was just, yknow, expecting something different.